addon/components/rdf-input-fields/concept-scheme-selector.hbs

```diff
@@ -1,10 +1,12 @@
-<AuLabel
-  @error={{this.hasErrors}}
-  @required={{this.isRequired}}
-  for={{this.inputId}}
->
-  {{@field.label}}
-</AuLabel>
+{{#unless @isTable}}
```

Comment:
At the moment I'm reusing the existing form fields by passing an extra argument into it. They can then decide how they should be displayed if rendered in a table. I've only done it for a couple of components. Not sure if it would be better to create dedicated components or not.

Reply:
The argument name is also up-for-debate but didn't want to lose to much time on it.

Reply:
I am also inclined that 's it is going to be messy once we extend features.